### PR TITLE
add Volume Health Monitor section

### PR DESCRIPTION
**Pull request by JacobGros**
# Description
This PR adds documentation for the new volume health monitor feature in CSI-PowerFlex
# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/84|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/v1/features/powerflex.md [hunk 1]
```diff
@@ -497,6 +497,26 @@ or you could edit logConfig.yaml, and use this command:
 `kubectl apply -f logConfig.yaml`  
 
 and then make the necessary adjustments for LOG_LEVEL and LOG_FORMAT.  
-If LOG_LEVEL or LOG_FORMAT are set to options outside of what is supported, the driver will use the default values of "info" and "text" . 
+If LOG_LEVEL or LOG_FORMAT are set to options outside of what is supported, the driver will use the default values of "info" and "text" .
```
Comment (bharathsreekanth) [inline content/v1/features/powerflex.md:500]: Is there a space at the end?  thats the only change i see here.

File: content/v1/features/powerflex.md [hunk 2]
```diff
@@ -497,6 +497,26 @@ or you could edit logConfig.yaml, and use this command:
...
 
 and then make the necessary adjustments for LOG_LEVEL and LOG_FORMAT.  
-If LOG_LEVEL or LOG_FORMAT are set to options outside of what is supported, the driver will use the default values of "info" and "text" . 
+If LOG_LEVEL or LOG_FORMAT are set to options outside of what is supported, the driver will use the default values of "info" and "text" .   
 
+## Volume Health Monitoring 
 
+Starting in version 2.1, CSI Driver for PowerFlex now supports volume health monitoring. This allows Kubernetes to report on the condition of the underlying volumes via events when a volume condition is abnormal. For example, if a volume were to be deleted from the array, or unmounted outside of Kubernetes, Kubernetes will now report these abnormal conditions as events.  
+
+To accomplish this, the driver utilizes the sidecar, external-health-monitor. When driver detects a volume condition is abnormal, the sidecar will report an event to the corresponding PVC. For example, in this event from `kubectl describe pvc -n <ns>` we can see that the underlying volume was deleted from the PowerFlex array:
```
Comment (bharathsreekanth) [inline content/v1/features/powerflex.md:506]: I would rephrase the first sentence as "To accomplish this, the driver utilizes the external-health-monitor sidecar"

**Review by bharathsreekanth — DISMISSED** [review]

**Review by yifeijin — APPROVED** [review]

File: content/v1/features/powerflex.md (hunk 1 above)
Comment (JacobGros) [reply content/v1/features/powerflex.md:500]: I think it's showing that I added a new line

File: content/v1/features/powerflex.md (hunk 2 above)
Comment (JacobGros) [reply content/v1/features/powerflex.md]: done

**Review by bharathsreekanth — APPROVED** [review]

**Review by jooseppi-luna — APPROVED** [review]
